web_poet/testing/pytest.py

```diff
@@ -0,0 +1,100 @@
+import asyncio
+import json
+import operator
+from pathlib import Path
+from typing import Any, Iterable, List, Optional, Set, Union
+
+import pytest
+
+from web_poet import ItemPage
+from web_poet.serialization import SerializedDataFileStorage, deserialize, load_class
+from web_poet.testing import INPUT_DIR_NAME, OUTPUT_FILE_NAME
+from web_poet.utils import ensure_awaitable
+
+
+class WebPoetFile(pytest.File):
+    """Represents a directory containing test case subdirectories for one page object."""
+
+    @staticmethod
+    def sorted(items: List["WebPoetItem"]) -> List["WebPoetItem"]:
+        """Sort the item list by the item name."""
+        return sorted(items, key=operator.attrgetter("name"))
+
+    def collect(self) -> Iterable[Union[pytest.Item, pytest.Collector]]:  # noqa: D102
+        result: List[WebPoetItem] = []
+        for entry in self.path.iterdir():
+            if entry.is_dir():
+                item: WebPoetItem = WebPoetItem.from_parent(
+                    self,
+                    name=entry.name,
+                    path=entry,
+                )
+                if item.input_path.is_dir() and item.output_path.is_file():
+                    result.append(item)
+        return self.sorted(result)
+
+
+class WebPoetItem(pytest.Item):
+    """Represents a directory containing one test case."""
+
+    @property
+    def po_name(self) -> str:
```

Comment:
Could you please rename this method (and similar methods) to avoid using "po" shorcut? We often use "page" in web-poet, but other names might be more appropriate. The same applies to docstrings.